hal/halx86/acpi/madt.c

```diff
@@ -42,11 +42,12 @@ HALP_APIC_INFO_TABLE HalpApicInfoTable;
 // Bits 2-31 are reserved.
 
 static PROCESSOR_IDENTITY HalpStaticProcessorIdentity[MAXIMUM_PROCESSORS];
-const PPROCESSOR_IDENTITY HalpProcessorIdentity = HalpStaticProcessorIdentity;
+PPROCESSOR_IDENTITY HalpProcessorIdentity;
```

Comment:
Why do we need this pointer to not be set at compile time to the static buffer above?

Reply:
he just changed this for no reason, so I changed it back.

Reply:
What's the point of this pointer at all?

Reply:
I actually don't see one, so i just entirely removed it